### PR TITLE
Feature/tauane branch

### PR DESCRIPTION
**Pull request by tauanesales**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                </div>
+            </div>
+        </section>
+        <hr class="m-0" />
+
+    </div>
+    <!-- Bootstrap core JS-->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+    <!-- Core theme JS-->
+    <script src="js/scripts.js"></script>
```
Comment (ArielMAJ) [inline index.html:76]: The file path and actual file names are different. Please update this.

**Review by ArielMAJ — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (tauanesales) [reply index.html:76]: Done. Thanks for the review!

Comment (ArielMAJ) [reply index.html:76]: Perfect!

**Review by ArielMAJ — APPROVED** [review]
LGTM!

**Pull request merged**